### PR TITLE
Adds information on writing CorDapps. Reorg.

### PR DESCRIPTION
**Pull request by joeldudleyr3**

### CODE REVIEW

File: docs/source/running-a-node.rst [hunk 1]
```diff
@@ -3,9 +3,8 @@ Running a node
 
 Starting your node
 ------------------
-
 After following the steps in :doc:`deploying-a-node`, you should have deployed your node(s) with any chosen CorDapps
-installed. You run each node by navigating to ``<node_dir>`` in a terminal window and running:
+already installed. You run each node by navigating to ``<node_dir>`` in a terminal window and running:
 
 .. code-block:: shell
```
Comment (ragmondo) [inline docs/source/running-a-node.rst]: I know it's not your change, but is this strictly true - on unix you don't need to prefix with java -jar ?

File: docs/source/deploying-a-node.rst [hunk 2]
```diff
@@ -1,48 +1,12 @@
 Deploying a node
 ================
 
-Building nodes using Gradle
+Using Gradle to build nodes
```
Comment (ragmondo) [inline docs/source/deploying-a-node.rst]: lol at this change

File: docs/source/running-a-node.rst [hunk 3]
```diff
@@ -22,25 +21,8 @@ Otherwise the workspace folder for the node is the current working path.
...
 Debugging your node
 -------------------
-
 To enable remote debugging of the node, run the following from the terminal window:
 
 ``java -Dcapsule.jvm.args="-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005" -jar corda.jar``
 
-This command line will start the debugger on port 5005 and pause the process awaiting debugger attachment.
-
-Viewing the persisted state of your node
```
Comment (ragmondo) [inline docs/source/running-a-node.rst]: Where did the database stuff go ? Is this duplicated on another section now ?

Comment (joeldudleyr3) [inline docs/source/running-a-node.rst]: Yep - there's a `node database` section.

File: docs/source/running-a-node.rst (hunk 1 above)
Comment (joeldudleyr3) [inline docs/source/running-a-node.rst]: Thanks, addressed.

Comment (joeldudleyr3) [inline docs/source/running-a-node.rst]: Fixed, thanks

**Pull request merged**